readme.md

```diff
@@ -295,133 +292,14 @@ session-endpoint = "https://connect.gataca.io:9090/admin/v1/login/gataca">
 </html>
 ````
 
-___ Note: ___  _The following examples explain how to use certain configuration parameters depending on your application needs_
+You can use this component with an already created session, which can be inserted on the sessionId property on the element, or passed via query parameter _id_ or _sessionId_ on the current URL.
+You can also provide a method to generate a new session like in the example, or, in the rare event of matching the authorizer API, just the endpoint to your application.
 
-#### Application rendering HTML Only
-This example shows how to integrate the QR Component on a normal scenario, where the application defines its own interface for the services.
-
-You can find an example of that kind of simple application _(written in Go)_ on the [Gataca Authorizer](https://github.com/gatacaid/gataca-authorizer), which we will use as example to explain the component's usage. 
-*Gataca Authorizer* creates the sessionId before rendering the page and provides it as a parameter on the HTML template. To query the status of the session, it offers the following endpoint:
-
-* **/login/status** : _Check the status of the created session_
-
-Continuing with the example as before, you could integrate with that kind of application - _running at $APP_DOMAIN_- by modifying the following pieces of code
-
-1. Add JQuery Library in HEAD
-````html
-<head>
-...
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-...
-</head>
-`````
-
-2. Modify the parameters of the QR to present the connect server and the session id renderized on the HTML template when loaded by the backend.
-````html
-<body>
-  ...
-  <gataca-qr id="gataca-qr" callback-server="{{.ConnectServer}}" session-id="{{.SessionId}}">
-  ...
-</body>
-`````
-
-3. Include a checkStatus implementation to retrieve the session data.
```

Comment:
Why do you remove this part? I thnk people needs an example to integrate it.

Reply:
I think it's done because of merge conflicts, same as the reverting the version